docs/OPTIONS.md

```diff
@@ -40,20 +41,44 @@ remote_compression_level={LEVEL}
  
 ## Ignore Rule Files
 
+To speedup execution you might want to **ignore large directories and files** not required for remote command such as `/.git` and so on. 
```

Comment:
Large directories and files ignoring can speedup sync from and to remote machine.